ee/docker-ee/ubuntu.md

```diff
@@ -13,83 +13,62 @@ toc_max: 4
 
 >{% include enterprise_label_shortform.md %}
 
-To get started with Docker Engine - Enterprise on Ubuntu, make sure you
-[meet the prerequisites](#prerequisites), then
-[install Docker](#install-docker-ee).
-
-## Prerequisites
-
-Docker Engine - Community users should go to
+> **Important** 
+> 
+> Docker Engine - Community users should go to
 [Get Docker Engine - Community for Ubuntu](/install/linux/docker-ce/ubuntu.md)
-**instead of this topic**.
+**instead of this topic**. Confirm that all prerequisites are met before installing Docker Engine - Enterprise on Ubuntu
+{: .important}
 
-To install Docker Engine - Enterprise, you need to know the Docker Engine -
-Enterprise repository URL associated with your trial or subscription. These
-instructions work for Docker on Ubuntu and for Docker on Linux, which includes
-access to Docker Engine - Enterprise for all Linux distributions. To get this
-information, do the following:
+The product URL associated with your trial or subscription is required to install Docker Engine - Enterprise, which can be attained using the following procedure (note that these instructions apply to Docker on Ubuntu and for Docker on Linux, which includes access to Docker Engine - Enterprise for all Linux distributions):
 
-- Go to [https://hub.docker.com/my-content](https://hub.docker.com/my-content).
-- Each subscription or trial you have access to is listed. Click the **Setup**
+1. Go to [https://hub.docker.com/my-content](https://hub.docker.com/my-content).
+2. Each subscription or trial you have access to is listed. Click the **Setup**
   button for **Docker Enterprise Edition for Ubuntu**.
-- Copy the URL from the field labeled
+3. Copy the URL from the field labeled
   **Copy and paste this URL to download your Edition**.
 
 Use this URL when you see the placeholder text `<DOCKER-EE-URL>`.
 
-To learn more about Docker Enterprise, see
+To learn more about Docker Engine - Enterprise, see
 [Docker Enterprise Edition](https://www.docker.com/enterprise-edition/){: target="_blank" class="_" }.
 
 ### OS requirements
 
 To learn more about software requirements and supported storage drivers,
 check the [compatibility matrix](https://success.docker.com/article/compatibility-matrix).
 
-> **Note:**
-> IBM Z (`s390x`) is supported for Docker Engine - Enterprise 17.06.xx only.
+> **Note**
+> 
+> IBM Z (`s390x`) is supported for Docker Engine - Enterprise 17.04.xx only.
```

Comment:
Was changing `17.06` to `17.04` intentional? I don't think we ever shipped a 17.04 enterprise engine

Reply:
Fixed